app/pages/view-application.tsx

```diff
@@ -0,0 +1,59 @@
+import React, {Component} from 'react';
```

Comment:
Looks like this is now a duplicate of the `applications-details` page?

Reply:
Except for the "Back" button maybe. Which reminds me that we should probably use the `DefaultLayout`'s `showSubheader` prop instead

Reply:
It is similar to Summary page (`ApplicationWizardConfirmation`) but doesn't have Continue button.

Reply:
The `ApplicationWizardConfirmation` got refactored into an `ApplicationDetails` container, which doesn't have that button. That's being rendered by the `application-details` page at the moment, so I'm not sure the `view-application` page is needed anymore

Reply:
Ok let me see if i can reuse `application-details` page instead.

Reply:
I checked, and `application-details` cannot be reused. It's a different page.

Reply:
Right, there's probably some naming issues here. Can you at least use the `ApplicationDetails` component instead of iterating on the `formResultsByApplicationId` (`ApplicationDetails` should be doing that for you)

Reply:
Yeap, I added `ApplicationDetails` component. Updated tests.